cavern/src/main/java/org/opencadc/cavern/files/CavernURLGenerator.java

```diff
@@ -308,78 +317,50 @@ private List<URI> handleContainerMount(VOSURI target, ContainerNode node, Protoc
         return ret;
     }
 
-    public VOSURI getNodeURI(String meta, String sig, Direction direction) throws AccessControlException, IOException {
+    public VOSURI getNodeURI(String token, VOSURI targetVOSURI, Direction direction) throws AccessControlException, IOException {
```

Comment:
I wonder if this should be called validate() or something.

Reply:
changed it to:

public void validateToken(String token, VOSURI targetVOSURI, Direction direction)